### ISSUE TITLE
Flag video privacy level in individual responses page

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#393]
**TL;DR**: Prominently show the privacy level of videos on the individual responses page.

**Narrative**
"As an Experimenter, I sometimes want to quickly get an example video from a study to share with a colleague or in a talk. It is very important that I do not screw up and select one that should not be shared in the relevant setting."

**Acceptance Criteria**
- [ ] QA on new feature: Private, scientific, public levels are each displayed appropriately. If session is withdrawn, that is displayed instead. If no privacy level was selected, either "private" or "private (no privacy level selected)" is displayed.
- [ ] Test by at least one beta-tester
- [ ] Scoping completed for more robust solution e.g. adding privacy level to videoname upon download; separate issue created if appropriate.

**Implementation Notes**
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

**Comment by kimberscott** [comment]
Largely covered by recent cleanup of individual responses page, although could always use more tests. Closing for now, can consider other additions in the future.
